test/benchee/formatters/html_test.exs

```diff
@@ -182,6 +182,39 @@ defmodule Benchee.Formatters.HTMLTest do
     end
   end
 
+  test "assets should not be copied when assets inlining is on" do
+    try do
```

Comment:
yes, this try is only for after because we want to be sure that files are removed despite potential errors in test body. Could you point me docs where this is described because i have never heard about this?